components/service/pocket/src/test/java/mozilla/components/service/pocket/ext/MappersKtTest.kt

```diff
@@ -71,4 +71,32 @@ class MappersKtTest {
         assertSame(story.url, result.url)
         assertSame(story.timesShown, result.timesShown)
     }
+
+    @Test
+    fun `GIVEN an internet spoc WHEN it is converted to a local spoc THEN a one to one mapping is made`() {
```

Comment:
nit: maybe GIVEN a spoc fetched from the network?

Reply:
Thank you.
Refactored to "GIVEN a spoc downloaded from Internet ..."